src/llama_recipes/utils/dataset_utils.py

```diff
@@ -54,6 +53,7 @@ def get_custom_dataset(dataset_config, tokenizer, split: str):
     "grammar_dataset": get_grammar_dataset,
     "samsum_dataset": get_samsum_dataset,
     "custom_dataset": get_custom_dataset,
+    "llamaguard_dataset": get_custom_dataset,
```

Comment:
Probably will be apparent when running it, but the function should be called differently from the "custom" dataset.